rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/fleet_adapter.py

```diff
@@ -42,325 +38,15 @@
 from rclpy.qos import qos_profile_system_default
 
 from .RobotCommandHandle import RobotCommandHandle
```

Comment:
This crashes at startup with a module not found error because the file does not exist anymore, it should probably be removed

Reply:
Whoops, fixed here: [ad48b8a](https://github.com/open-rmf/rmf_demos/pull/188/commits/ad48b8af12988576bcc8af8072dc1955f77a4eba)